vignettes/get_slide_data.R

```diff
@@ -128,6 +128,7 @@ names(sldat) <- capitalize(names(sldat))
 # Create links
 # sldat$PDF <- paste0("[pdf](https://github.com/compstat-lmu/lecture_i2ml/blob/master/slides-pdf/", sldat$Deck, ".pdf)")
 sldat$PDF <- paste0("[pdf](../../slides-pdf/", sldat$Deck, ".pdf)")
```

Comment:
will these links work online as well? 
I used these nbviewer links so that the pdf slides are rendered nicely directly in the browser...

Reply:
I thought they would. But I can't test this now.

Reply:
In the end, we know your solution works, so let's switch again to that.

Reply:
```suggestion
sldat$PDF <- paste0("[pdf](https://github.com/compstat-lmu/lecture_i2ml/blob/master/slides-pdf/", sldat$Deck, ".pdf)")
```